### DISCUSSION TITLE
Enrichir l'affiche?

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1178] [Idées]
Une utilisatrice (L. A.) demandait à ce que les logos qui apparaissent sur le profil d'une cantine quand elle en remplit les critères (cf. PJ) puissent aussi apparaître sur l'affiche de la cantine en plus des pourcentage d'appros. Qu'en pensez-vous?
<img width="67" alt="Capture d’écran 2022-03-14 à 14 27 56" src="https://user-images.githubusercontent.com/93709421/158181665-adc0ed48-ca31-431f-9d45-60926bcdab90.png">

**Comment by valeriemerle** [comment]
On pourrait proposer d'indiquer, le cas échéant, si une cantine recherche ses approvisionnements au sein d'un PAT (projet alimentaire de territoire)?
